qemu/tests/pxe_query_cpus.py

```diff
@@ -85,7 +85,15 @@ def run(test, params, env):
             count += 1
             try:
                 vm.monitor.info("cpus", debug=False)
-                vm.verify_status("running")
+                if params.get('machine_type').startswith("s390"):
+                    if 'running' in vm.monitor.get_status()['status'] or \
+                            'guest-panicked' in \
+                            vm.monitor.get_status()['status']:
+                        pass
+                    else:
+                        vm.verify_status("running")
```

Comment:
If the status is not `running` and not `guest-panicked`, this judgment will always fail, so can raise an error directly.